infra/cifuzz/cifuzz.py

```diff
@@ -320,24 +323,70 @@ def get_files_covered_by_target(latest_cov_info, target_name,
     return None
 
   # Cases like curl there is /src/curl and /src/curl_fuzzers/ are handled.
```

Comment:
Could you change this comment to some thing like "Make sure cases like /src/curl and /src/curl/ are both handled."